canonical-web-and-design/ubuntu.com
# Add `rel="noopener noreferrer"` to `target="_blank"` links

## Done

Links opening in a new tab (with attribute `target="_blank") are a potential vulnerability. 
More details and examples here: https://mathiasbynens.github.io/rel-noopener/
Adding `rel="noopener noreferrer"` fix that

## QA

- Open the demo site on a page which have a target="_blank" link such as [here](https://ubuntu-com-9146.demos.haus/wsl#:~:text=Discover%20how%20to%20easily%20access%20“Kubernetes%20on%20Windows%20with%20WSL2”%20in%20our%20webinar) 
- Verify that it still works as expected

## Issue / Card

Fixes #

## Screenshots

[If relevant, please include a screenshot.]
